### PR TITLE
Show last value on the legend, closes: #12

### PR DESCRIPTION
**Pull request by ppikula**
It is pretty static(especially the width), but I think we shouldn't
invest more time in hacking flot lib to display more advance stuff
we can just use: http://c3js.org/ It is very customizable and has
ready to use react component. The work will be tracked in issue: #31

### CODE REVIEW

File: priv/build/index.html [hunk 1]
```diff
@@ -54,9 +54,10 @@
      }
 
      .legend-label {
-       min-width: 400px !important;
-       padding: 10px;
-       margin-right: 10px;
+       font-size: 10px;
+       width: 100px !important;
```
Comment (afronski) [inline priv/build/index.html]: Do we have to apply `!important` here? There is no other way (e.g. specificity override)?

File: priv/app/graph_flot.jsx [hunk 2]
```diff
@@ -56,31 +57,50 @@ export default class FlotGraph  {
...
 
   close(data) {
   }
 
+  // "reactive" legend
+  updateLegendNoColumns() {
+    let opts = this.plot.getOptions();
+    let len = this.lines.length;
+    let w = this.plot.width();
+    let units = Math.floor(w/117);
```
Comment (afronski) [inline priv/app/graph_flot.jsx]: *Nitpicking*: A little bit *magic number* and no spaces around `/`.

File: priv/app/graph_flot.jsx [hunk 3]
```diff
@@ -56,31 +57,50 @@ export default class FlotGraph  {
         show: true,
       }
     });
+    this.updateLegendNoColumns();
   }
 
   labelFormatter(label, series) {
+    let dataPoint = series.data.slice(-1)[0][1];
```
Comment (afronski) [inline priv/app/graph_flot.jsx]: *Nitpicking*: A little bit *magic* here.

File: priv/app/graph_flot.jsx [hunk 4]
```diff
@@ -56,31 +57,50 @@ export default class FlotGraph  {
...
+    let val = Math.round(dataPoint/10.0)/100.0;
+    let unit = "ms";
+    if (label == "count"){
```
Comment (afronski) [inline priv/app/graph_flot.jsx]: *Nitpicking*: Missing space between `)` and `{`.

File: priv/app/graph_flot.jsx [hunk 5]
```diff
@@ -56,31 +57,50 @@ export default class FlotGraph  {
         show: true,
       }
     });
+    this.updateLegendNoColumns();
   }
 
   labelFormatter(label, series) {
+    let dataPoint = series.data.slice(-1)[0][1];
+    let val = Math.round(dataPoint/10.0)/100.0;
```
Comment (afronski) [inline priv/app/graph_flot.jsx]: *Nitpicking*: Missing spaces around operators.

Comment (ppikula) [inline priv/app/graph_flot.jsx]: Following the file "convention" ;)

File: priv/app/graph_flot.jsx (hunk 2 above)
Comment (ppikula) [inline priv/app/graph_flot.jsx]: will be removed with c3js

**Pull request merged**